src/SelectMulti.vue

```diff
@@ -406,7 +414,7 @@
 				:placeholder="placeholder"
 				:aria-placeholder="placeholder"
 				@blur="onInputBlur"
-				@click="updateMenuState(true)"
+				@click="updateMenuState(!open, !open)"
```

Comment:
I don't this it's a good solution. 
If we click on input it implies it's focused. and it means we need to have menu opened .
If we need to close menu on second input click - it's a wrong requirement (imo)

Reply:
Good point, I'll revert this change!

Reply:
Actually, sorry @ivan-shtyrlyaev-nitka  I misread this comment - I think it's standard and expected behavior for a second click to close the menu, and it would be confusing to the user if they didn't see that behavior. 
Here are recordings of a few examples I found - I wasn't able to find any example where a second click didn't close the menu:

**Google:**
https://github.com/user-attachments/assets/8e97b96b-e9ae-4884-bd0c-7ec3f4390472
**Outlook:**
https://github.com/user-attachments/assets/2f78f6b1-c4bb-48d5-a866-61231495e2a1
**Github:**
https://github.com/user-attachments/assets/79643eec-6123-4ee8-be6b-12cdc0f45105
**Jira:**
https://github.com/user-attachments/assets/72b79894-bd8b-4326-adae-4071ce32115e

Reply:
Thanks for explaining your thinking on this point, @ivan-shtyrlyaev-nitka ! You were right after all and I updated this PR to only support toggle behavior on the icon but not on the input. I also replicated these changes for `main` with this PR: https://github.com/politico/vue-accessible-selects/pull/147